todo_app/data/trello_items.py

```diff
@@ -0,0 +1,95 @@
+import requests,os, json
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
+
+key = os.getenv('TRELLO_API_KEY') 
+token = os.getenv('TRELLO_API_TOKEN')
+
+
+
+class Item:
+    def __init__(self, id, name, status, desc):
+        self.id = id
+        self.name = name
+        self.status = status
+        self.desc = desc 
+
+    @classmethod
+    def from_trello_card(cls, card, lists):
+        list_name = next((list['name'] for list in lists if list['id'] == card['idList']), None)
+        return cls(card['id'], card['name'], list_name, card.get('desc', ''))
+
+def get_board_id():
+    fields = 'fields=name'
+    url = 'https://api.trello.com/1/members/me/boards?{}&key={}&token={}'.format(fields,key,token)
+    reponse = requests.get(url)
+    boards_json = reponse.json()
+
+    for board in boards_json:
+        if board.get('name') =='DevOps Engineering':
+            board_id = board.get('id')
+    return board_id
+
+def get_lists():
+    fields = 'fields=name'
+    url = 'https://api.trello.com/1/boards/{}/lists?{}&key={}&token={}'.format(get_board_id(),fields,key,token)
+    reponse = requests.get(url)
+    lists_json = reponse.json()
+    return lists_json
+
+
+def get_items():
+    items = []
+    fields = 'fields=name,idList,desc,labels'
+    url = 'https://api.trello.com/1/boards/{}/cards?{}&key={}&token={}'.format(get_board_id(),fields,key,token)
+    reponse = requests.get(url)
+    cards_json = reponse.json()
+
+    for card in cards_json:
+        for list in get_lists():
+            if card.get('idList') == list.get('id'):
+                card.update({'idList':list.get('name')})
+                id = card.get('id')
+                status = card.get('idList')
+                title = card.get('name')
+                description = card.get('desc')
+                     
+                items.append({'id': id, 'status': status, 'title' : title, 'description':description})
```

Comment:
I'd be inclined to use your class directly here, and use this instead of the get_cards function so that we can reduce the code considerably, for example:

1. this section can be reduced to:
```python
            if card.get('idList') == list.get('id'):
                list_name = list.get('name')                
                items.append(Item.from_trello_card(card, list_name))
```
2. Which relies on a simplification to the Item class:
```python
    @classmethod
    def from_trello_card(cls, card, list_name):
        return cls(card['id'], card['name'], list_name, card.get('desc', ''))
```
3. And enables a simpler index:
```python
@app.route('/', methods=['GET','POST'] )
def index():
        items = get_items()
        return render_template ('index.html', my_items = items)
```
4. This does rely on changing a few other parts of the code, notably in `save_item` using the class version of the item, so e.g. `item.id` instead of `item.get('id')`, and `item.status = dict(request.form.items())['status']` instead of `item.update({'status': dict(request.form.items())['status']})`